go/vt/vtgate/planbuilder/gen4_planner.go

```diff
@@ -30,6 +30,7 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/vindexes"
 )
 
+//goland:noinspection GoVarAndConstTypeMayBeOmitted
```

Comment:
Do we use editor specific annotations in other places as well or should this be dropped?